streaming/java/streaming-api/src/main/java/io/ray/streaming/api/function/Function.java

```diff
@@ -7,4 +7,12 @@
  */
 public interface Function extends Serializable {
 
+  default void loadCheckpoint(Object checkpointObject) {
```

Comment:
It's hard to know what `checkpointObject` is if it's an `Obejct` type. Why use `Object` here. I think a custom type can give mo dfetailed explaintation

Reply:
Changed to Serializable